.gitignore

```diff
@@ -2,3 +2,6 @@
 target/
 dbt_packages/
 logs/
+dbt_project.yml
```

Comment:
@dgitis I don't think we want to ignore the dbt_project.yml or packages.yml .

dbt_project.yml may have useful materialization defaults. packages.yml may define may define dependencies for our package

Reply:
Definitely right about this. I'll fix it.